zc_install/sql/demo/demo_cleanup.sql

```diff
@@ -37,4 +37,5 @@ TRUNCATE TABLE record_company_info;
 TRUNCATE TABLE reviews; 
 TRUNCATE TABLE reviews_description; 
 TRUNCATE TABLE salemaker_sales; 
-TRUNCATE TABLE specials; 
+TRUNCATE TABLE specials;
+TRUNCATE TABLE featured_categories; 
```

Comment:
Each of the table truncations in this file are alphabetically ordered based on the table's name.